python-menus-toolbars/MIT.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Prahlad Yeri
```

Comment:
@lpozo Shouldn't this be your name instead? :slightly_smiling_face:

Reply:
Ohh, sorry about that. Copy/Paste is a bad practice.